framework/src/Volo.Abp.UI/Volo.Abp.UI.csproj

```diff
@@ -22,5 +22,12 @@
   <ItemGroup>
     <ProjectReference Include="..\Volo.Abp.Localization\Volo.Abp.Localization.csproj" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Update="Fody" Version="6.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
We can remove these because it is already configured in `configureawait.props`.

Reply:
Resolved apologies